.github/workflows/codespell.yml

```diff
@@ -10,5 +10,7 @@ jobs:
   codespell:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - uses: codespell-project/actions-codespell@v1.0
```

Comment:
Probably better to add `Manuel` to the `ignore-words-list` in the `.codespellrc` file, so that this warning is squelched for anyone running codespell locally too.

Reply:
Oops, I missed the `.codespellrc`. Done now.

Reply:
Thanks!

Reply:
Seems to need to be not capitalised